Example/Production/govuk_onboarding_ios/SupportingFiles/SceneDelegate.swift

```diff
@@ -1,8 +1,8 @@
 //
-//  SceneDelegate.swift
-//  govuk-mobile-ios-onboarding
+// GOV.UK
 //
-//  Created by Thomas Bates on 18/04/2024.
+// Copyright © 2024 Government Digital Services.
+// All Rights Reserved.
 //
```

Comment:
As [mentioned](https://github.com/alphagov/govuk-mobile-ios-services/pull/3#discussion_r1579135241) - not something we'd normally do.